vagrant-libvirt.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'fog-libvirt', '>= 0.3.0'
   s.add_runtime_dependency 'nokogiri', '~> 1.6.0'
-  s.add_runtime_dependency 'fog-core', '~> 1.43.0'
+  s.add_runtime_dependency 'fog-core', '~> 2.1.0'
```

Comment:
I'm not 100% sure what the impact of this change would be, so far it looks like it adds a dependency on ruby >= 2.0, but as vagrant 1.8 used ruby 2.2 I'm not sure that is a problem.